winsup/cygwin/include/cygwin/version.h

```diff
@@ -474,13 +474,12 @@ details. */
   307: Export timingsafe_bcmp, timingsafe_memcmp.
   308: Export dladdr.
   309: Export getloadavg.
-  310: Export kill_process_tree.
 
   Note that we forgot to bump the api for ualarm, strtoll, strtoull,
   sigaltstack, sethostname. */
 
 #define CYGWIN_VERSION_API_MAJOR 0
-#define CYGWIN_VERSION_API_MINOR 310
+#define CYGWIN_VERSION_API_MINOR 309
```

Comment:
Yes, that's because this is a `fixup!` commit that reverts an earlier change that increased it.

I added `kill_process_tree` to the exported functions, and that requires the minor number to be incremented. But I no longer need that function, nor any other exported function.

So yes, this decrement is intentional.